addons/web_planner/tests/test_web_planner.py

```diff
@@ -26,10 +27,9 @@ def test_00(self):
             }
         )
 
-        self.assertEqual(
-            Planner.prepare_backend_url(self, "web_planner.test_00"),
-            "/web#view_type=list&action=%s" % action.id,
-        )
+        backend_url = Planner.prepare_backend_url(self, "web_planner.test_00", 'form')
```

Comment:
```suggestion
        backend_url = Planner.prepare_backend_url(self, "web_planner.test_00")
```
you should remove the 'form' in this line. The first test tests that preapre_backend_url returns the correct backend url with the default view_type when only the action_xml_id is given.